NativeApp/src/components/Login/Form/index.js

```diff
@@ -56,23 +57,24 @@ class LoginForm extends Component {
           onChangeText={text => this.setState({ password: text })}
           value={password}
           secureTextEntry
-          underlineColorAndroid={hasError ? 'red' : underlineColor2}
+          underlineColorAndroid="transparent"
+          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor2 }}
           selectionColor="#00DCFA"
         />
         {
           hasError && (
-            <Text validationText>
+            <Caption>
               Incorrect email or password
-            </Text>
+            </Caption>
           )
         }
-        <TouchableOpacity
+        <Button
```

Comment:
Think we should use react-native-elements button instead for the buttons.

Reply:
made the changes :)